frontend/packages/gitops-plugin/src/components/list/GitOpsListItem.tsx

```diff
@@ -2,32 +2,41 @@ import * as React from 'react';
 import * as _ from 'lodash';
 import { useTranslation } from 'react-i18next';
 import { Grid, GridItem, Card, CardBody } from '@patternfly/react-core';
-import { history, ResourceLink } from '@console/internal/components/utils';
+import { history, ResourceLink, ExternalLink } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
 import { GitOpsAppGroupData } from '../utils/gitops-types';
+import { getArgoCDFilteredAppsURI } from '../utils/gitops-utils';
 import './GitOpsListItem.scss';
 
 interface GitOpsListItemProps {
   appGroup: GitOpsAppGroupData;
+  argocdLink?: K8sResourceKind;
 }
 
-const GitOpsListItem: React.FC<GitOpsListItemProps> = ({ appGroup }) => {
+const GitOpsListItem: React.FC<GitOpsListItemProps> = ({ appGroup, argocdLink }) => {
   const { t } = useTranslation();
+  const argocdLinkUrl = getArgoCDFilteredAppsURI(argocdLink.spec.href, appGroup.name);
   const handleCardClick = () => {
     history.push(`/environments/${appGroup.name}?url=${appGroup.repo_url}`);
   };
 
   return (
-    <Card className="odc-gitops-list-item" onClick={handleCardClick} isHoverable>
+    <Card className="odc-gitops-list-item" isHoverable>
```

Comment:
Whole card is not clickable anymore because of moving the onClick handler to two grid items only. I know you did the change because the click on argocd link would also result in clicking the card but I think a better way to solve this would be to stop propagation of event in `ExternalLink` component.

Reply:
Thanks Rohit.